lib/guardian_db.ex

```diff
@@ -1,11 +1,11 @@
-defmodule GuardianDb do
+defmodule GuardianDB do
```

Comment:
@yordis please change this back to `GuardianDb` so we can move forward with your PR.

This can be discuss again at a later date but at this time offers no value to the project.